playground/backend/containers/python/Dockerfile

```diff
@@ -49,7 +49,6 @@ COPY --from=build /go/bin/server_python_backend /opt/playground/backend/
 COPY --from=build /go/src/playground/backend/configs /opt/playground/backend/configs/
 
 # Install Python Katas Utils
-COPY katas/log_elements.py /go/src/katas/
```

Comment:
A lot of playground examples have:

`from log_elements import LogElements`. These statements would stop working, so we should either look how to replace them or for a drop-in replacement keep katas/log_elements.py which will simply import LogElements from util.py

Reply:
I apologize in advance if my understanding is wrong, but wouldn't the changes in the kata files (such as [this one](https://github.com/apache/beam/blob/ed1a98194d06f75b0bccb9bcd31c975303f388ad/learning/katas/python/Introduction/Hello%20Beam/Hello%20Beam/task.py#L31)) correct this problem, as the playground backend would pull the updated Python SDK with the changes there after the merge?

Reply:
ah, I think you are right, I didn't realize playground just re-used code from katas.

Reply:
would using beam.LogElements work instead? that sounds like it would make katas code more uniformly looking

Reply:
The beam.LogElements import does work on the tests I ran, and it's definitely a cleaner way to import the transform. I made a new commit with this update.